go.sum

```diff
@@ -115,6 +115,7 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/toml v0.4.1 h1:GaI7EiDXDRfa8VshkTj7Fym7ha+y8/XxIgD2okUIjLw=
```

Comment:
Can you remove this `go.sum` update from the pull-request?
I think it may be because you have extra/past work done in your directory already which is not yet merged to main or sticked into your `go.sum`? Not sure. But this is probably better to not commit it as-is as there is many deps changes in there unrelated to your actual fix.